content/ja/docs/concepts/configuration/overview.md

```diff
@@ -77,7 +77,7 @@ services)（`ClusterIP`が`None`）を使用してServiceを簡単に検出で
 
 - `imagePullPolicy` のタグが省略されていて、利用してるイメージのタグが`:latest`の場合や省略されいてる場合、`Always`が適用されます。
```

Comment:
```suggestion
- `imagePullPolicy` のタグが省略されていて、利用してるイメージのタグが`:latest`の場合や省略されている場合、`Always`が適用されます。
```

Reply:
Can you also fix this too? I've just noticed 😛

Reply:
Sure!